xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/RecentlyUpdated.xml

```diff
@@ -204,19 +209,19 @@ require(['jquery'], $ =&gt; {
         const hasNext = JSON.parse($dataElement.attr('data-has-next'));
         if (!hasNext) {
           $macro.find('button.show-more')
-            .replaceWith('&lt;div class="no-more-results"&gt;No more results&lt;/div&gt;')
+            .replaceWith('&lt;div class="no-more-results"&gt;' + l10n['noMoreResults'] + '&lt;/div&gt;')
```

Comment:
Missing XML escaping. Maybe:

```suggestion
            .replaceWith('&lt;div class="no-more-results"&gt;&lt;/div&gt;').text(l10n['noMoreResults'])
```

Reply:
`l10n['noMoreResults']` is now already escaped from it's definition, there is no need to escape it twice.